pycds/orm/tables.py

```diff
@@ -276,8 +293,15 @@ class Variable(Base):
     network_id = Column(Integer, ForeignKey("meta_network.network_id"))
 
     # Relationships
-    network = relationship("Network", backref=backref("meta_vars", order_by=id))
-    obs = relationship("Obs", backref=backref("meta_vars", order_by=id))
+    network = relationship("Network", back_populates="variables")
+    meta_network = synonym("network")
+    obs = relationship("Obs", order_by="Obs.id", back_populates="variable")
```

Comment:
Is the `order_by` strictly necessary here? It's possibly that ordering on a huge table might add unnecessary computation. Plus nearly all of the queries that include `obs_raw` are going to order by the datetime of observation.

Reply:
Just copying what was in the original code, but good point. Will remove.